sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -958,6 +957,8 @@
   public static final String TELEMETRY_EDATA_USER_MERGE_ACTION = "merge-user-courses-and-cert";
   public static final String BE_JOB_REQUEST = "BE_JOB_REQUEST";
   public static final String TELEMETRY_ACTOR_USER_MERGE_ID = "Merge User Courses and Cert";
-  public static final String SUNBIRD_ACCOUNT_MERGE_BODY  = "sunbird_account_merge_body";
+  public static final String CERTIFICATE = "Certificate";
+  public static final String MERGE_CERT = "Mergecert";
```

Comment:
Mergecert to MergeCert

Reply:
We are using this in telemetry, there only first character is captial and all remaining is lower-case